src/sbvr-api/permissions.ts

```diff
@@ -692,26 +692,24 @@ export const checkPassword = (
 }> =>
 	getCheckPasswordQuery()({
 		username,
-	}).then((result: AnyObject[]) => {
+	}).then(async (result: AnyObject[]) => {
```

Comment:
```suggestion
	}).then(async ([user]: AnyObject[]) => {
```
and we can stop repeating `result[0]`

Reply:
Done